src/store/messages/saga.ts

```diff
@@ -196,6 +201,7 @@ export function* send(action) {
 
   yield call(uploadFileMessages, channelId, rootMessageId, uploadableFiles);
   yield call(publishMessageSent);
+  yield call(publishUserStoppedTypingEvent, channelId);
```

Comment:
Small note, I think it probably would have been cleaner to listen for the chatMessageBus event in the other saga file. That way all the typing stuff is together.

Reply:
makes sense! Updated.